### PR TITLE
Follow-up meeting - 04.10.2018

### PR DESCRIPTION
**Pull request by daneos**
I have created the basic agenda with several points I think we should cover during the brainstorming and the rest of the meeting as well. This list is by no means complete, so feel free to add more.

Sorry I'm adding this so late, I was busy getting beamlines ready for our first users today :)

### CODE REVIEW

File: 20181004/AGENDA.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+To be held on 2018/10/04 at 14:00  
+This one will be a bit unusual, since we will focus more on improving our work
+
+# Agenda
+1. Quick overview of urgent issues
+	* Alternative to appear.in
+	* Current development problems
+	* New urgent issues (if any)
+2. Brainstorming
```
Comment (reszelaz) [inline 20181004/AGENDA.md:10]: Thanks for posting the brainstorming ideas. I let myself to add more.. but it is not mandatory to add yours now. We will have time on the brainstorming to catch them...
First of all I see like two categories here. First are the objectives or topics to improve, feedback on.. and then are the tools. So propose to talk about the following objectives:
* follow-up meetings
* issues priorization/triage
* colaborative development
* communication with users
* releases

And the tools that could be used to improve these objectives:
* video conferences tools
* teams discussions
* team collaborative tools e.g. slack
* project boards
* issues labels
* milestones
* newsletter
* questioners

**Comment by reszelaz** [comment]
For me the minutes are ok!

**Comment by teresanunez** [comment]
For me too, thanks @daneos

**Review by reszelaz — APPROVED** [review]

**Review by teresanunez — APPROVED** [review]

**Comment by daneos** [comment]
I updated the agenda to reflect the points covered during meeting. I think it can be merged.

**Pull request merged**